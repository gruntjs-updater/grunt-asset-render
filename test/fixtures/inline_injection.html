well here are some assets <!-- START --> test/fixtures/assets/test.css,  test/fixtures/assets/test.js,  test/fixtures/assets/testing.html, <!-- END -->. Awesome!