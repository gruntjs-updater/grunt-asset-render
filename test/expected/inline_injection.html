well here are some assets <!-- START --> test/fixtures/assets/test.css,  test/fixtures/assets/test.js, <!-- END -->. Awesome!